### PR TITLE
docs: Access Control API Reference

### LINKED ISSUE #223: `POST`: Create signing keys

### PR DESCRIPTION
**Pull request by ichung08**
resolves #223 #226 #227 #222 #225 #206

### CODE REVIEW

File: docs/05-reference/api/create-asset-with-url.mdx [hunk 1]
```diff
@@ -32,9 +32,9 @@ curl --location --request POST 'https://livepeer.studio/api/asset/import' \
 
 ```js
 const response = await fetch("https://livepeer.studio/api/asset/upload/url", {
+    method: "POST",
     headers: {
-      method: "POST"
-      Authorization: "Bearer {api_key}",
+      Authorization: ${api_key},
```
Comment (victorges) [inline docs/05-reference/api/create-asset-with-url.mdx:37]: This one was actually was correct before. Must have the `Bearer` prefix.
```suggestion
      Authorization: `Bearer ${api_key}`,
```

File: docs/05-reference/api/create-signing-keys.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: POST signing key
+---
+
+# `POST` Create a signing key
+
+A `signing key` is used to sign JWTs and are necessary to make a stream private.
+
+When you create a `signing key`, the request returns a ECDSA (P-256) `pem` keypair, where
+- The `publicKey` is a Base64 of the `pem` public key of the keypair
```
Comment (victorges) [inline docs/05-reference/api/create-signing-keys.mdx:10]:
```suggestion
- The `publicKey` is a representation of the public key, encoded as base 64. You do not need to worry about this but just pass it around as a string.
```

File: docs/05-reference/api/create-signing-keys.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- The `privateKey` is displayed *only* on creation. This is the only moment where the client can save the private key, otherwise it will be lost
+Upload a video `Asset` to Livepeer Studio from an external URL through the
+`POST /api/asset/import` API.
```
Comment (victorges) [inline docs/05-reference/api/create-signing-keys.mdx:13]: I think this was left here by mistake.
```suggestion
```

File: docs/05-reference/api/update-signing-key.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- `name`
+- `disabled`
+
+All fields are optional, and if any field is not included in the request payload
+it will be kept unchanged from the saved object.
+
+## Request
+
+### Disabling a signing key
+
```
Comment (victorges) [inline docs/05-reference/api/update-signing-key.mdx:18]: Can add some context here:
```suggestion

This is useful when you want to rotate signing keys but don't want to risk deleting the old one permanently. Instead you can disable it first and check that nothing broke to only then delete it permanently.

```

File: docs/05-reference/api/update-signing-key.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+--header 'Authorization: Bearer {api_key}' \
+  -X PATCH \
+  -H "Content-Type: application/json" \
+--data-raw '{
+    "disabled":true
+}'
+```
+
+### Changing signing key name
+
```
Comment (victorges) [inline docs/05-reference/api/update-signing-key.mdx:30]:
```suggestion

Notice that you can also provide a name to the signing key upon creation. Just provide the same `"name"` field on the creation API.

```

**Review by victorges — APPROVED** [review]
LGTM

**Review by adamsoffer — APPROVED** [review]
LGTM

**Pull request merged**